### PR TITLE
Fix attribution if copyright is set by user.

### PR DESCRIPTION
**Pull request by MoritzFago**

### CODE REVIEW

**Comment by panr** [comment]
Hi @MoritzFago. Thanks for this PR! 😎 

Often, first thing someone does is to delete this section from a theme, so I let users override the entire copyright section (because it's open source and on MIT license and so on)... ¯\_(ツ)_/¯ 

**...but** maybe you're right and I should leave an attribution by default... so maybe we should add optional param in config to `leaveCreatorCopyrights` or something? What do you think?

**Comment by mohanboddu** [comment]
I wanted to overwrite the entire copyright section. But it didn't work for me. I added

```
copyright = "All rights reserved - 2019"
```
to `config.toml` file. But its still showing the default one. Is there something I am missing?

**Comment by panr** [comment]
@mohanboddu weird... did you restart `hugo server` after that?

**Comment by MoritzFago** [comment]
> Often, first thing someone does is to delete this section from a theme, so I let users override the entire copyright section (because it's open source and on MIT license and so on)... ¯_(ツ)_/¯

Thats seems very user-friendly, but to use your work, I need a license of some kind and I have to comply to the terms of the license.
> ...but maybe you're right and I should leave an attribution by default... so maybe we should add optional param in config to leaveCreatorCopyrights or something? What do you think?

In this case, you went with the MIT-license. MIT requires attribution and a reproduction of the license text. So if I'd use terminal without mentioning you somewhere, I'd be in breach of license, which would void my license (also the license text is currently missing, I'd put it somewhere in the CSS. 
So in conclusion: If you make that configuareable, you are supporting your user to breach their license, which would be interessting if you were to sue them, but without that its just strange.
So either make it non configuareable or choose a license which doesn't require attribution like [Unlicense](https://choosealicense.com/licenses/unlicense/) or [CC0](https://choosealicense.com/licenses/cc0-1.0/) 

P.S I am not a lawyer.

**Comment by mohanboddu** [comment]
> @mohanboddu weird... did you restart `hugo server` after that?

Yes, I did. I also did some debugging using [template debugging](https://gohugo.io/templates/template-debugging/)
```
{{ printf "%#v" $.Site }}
```
But the copyright value from it returned empty.

I am no expert in hugo, so I might be missing something.

I am attaching my hugo test website with copyright value set but its not replacing the default copyright attribute.
[hugo_test.tar.gz](https://github.com/panr/hugo-theme-terminal/files/3001300/hugo_test.tar.gz)

Thanks for your quick reply and helping me out.

**Comment by mohanboddu** [comment]
> > @mohanboddu weird... did you restart `hugo server` after that?
> 
> @panr Yes, I did. I also did some debugging using [template debugging](https://gohugo.io/templates/template-debugging/)
> 
> ```
> {{ printf "%#v" $.Site }}
> ```
> But the copyright value from it returned empty.
> 
> I am no expert in hugo, so I might be missing something.
> 
> I am attaching my hugo test website with copyright value set but its not replacing the default copyright attribute.
> [hugo_test.tar.gz](https://github.com/panr/hugo-theme-terminal/files/3001300/hugo_test.tar.gz)
> 
> Thanks for your quick reply and helping me out.

Its my mistake again, I put it under the main hugo settings rather than the theme settings. So, I fixed it and is working fine.

Thanks.

File: layouts/partials/header.html [hunk 1]
```diff
@@ -3,9 +3,11 @@
     <div class="header__logo">
       {{ partial "logo.html" . }}
     </div>
-    <div class="menu-trigger">menu</div>
+      {{ range $.Site.Menus.foo }}
```
Comment (panr) [inline layouts/partials/header.html]: I'm sorry, what's this? 🤔

File: layouts/partials/logo.html [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <a href="{{ if $.Site.Params.Logo.LogoHomeLink }}{{ $.Site.Params.Logo.LogoHomeLink }}{{else}}{{ $.Site.BaseURL }}{{ end }}">
   <div class="logo">
+  	{{$.Site.Params.Logo.logoTextPre}}
```
Comment (panr) [inline layouts/partials/logo.html]: Or this?

File: layouts/partials/logo.html [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
     {{ with $.Site.Params.Logo.logoText }}{{ . }}{{ else }}Terminal{{ end }}
+    {{$.Site.Params.Logo.logoTextPost | safeHTML}}
```
Comment (panr) [inline layouts/partials/logo.html]: ...and this?

**Comment by panr** [comment]
I'm closing it, due to lack of response ;-(
